presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationSmokeTest.java

```diff
@@ -44,18 +50,17 @@
         extends AbstractTestIntegrationSmokeTest
 {
     private TestingKafka testingKafka;
-    private String topicName;
+    private String rawTopicName;
```

Comment:
what is a raw topic name? how does it differ to topic name?

Reply:
I can see how that might be confusing. It is meant to stand for the raw data format, I'll rename it.